content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -77,7 +114,7 @@ On this page, the focus is on the data model. You can learn more about [Data sou
 
 The data model is a collection of [models](../../components/prisma-schema/data-model#defining-models). A model has two major functions:
 
-- Represent a table in the underlying database
+- Represent a table or collection in the underlying database
```

Comment:
Use of "collection" in both contextes might be confusing to users

Reply:
Both contexts?

Reply:
Ah, this referred to the line before - 115. There is is "collection of models" and then "table of collection".